app/assets/javascripts/preferences/panes/account/subscription/CancelledSubscription.tsx

```diff
@@ -0,0 +1,50 @@
+import { observer } from 'mobx-react-lite';
+import { SubscriptionState } from './subscription_state';
+import { Text } from '@/preferences/components';
+import { Button } from '@/components/Button';
+
+type Props = {
+  subscriptionState: SubscriptionState;
+};
+
+export const CancelledSubscription = observer(
```

Comment:
Seems this and `ActiveSubscription` components are very similar, does it make sense to combine them into one component and render respective parts depending on props?

Reply:
Done! Refactored them into one component